extensions/guacamole-auth-cas/src/main/java/org/apache/guacamole/auth/cas/ticket/TicketValidationService.java

```diff
@@ -81,14 +93,13 @@
      *     password values in.
      *
      * @return
-     *     A Map all of tokens for the user parsed from attributes returned
-     *     by the CAS server.
+     *     The CASAuthenticatedUser instance returned by the CAS server.
```

Comment:
I think this needs to be clarified. The CAS server is not returning an instance of the `CASAuthenticatedUser` class to us.

Reply:
Latest commits should resolve this and an instance of CASAuthenticatedUser should be returned now

Reply:
I'm not sure this actually addresses Mike's comment, here.  Mike is pointing out that it is inaccurate to say that the "CASAuthenticatedUser instance returned by the CAS server", because the CAS server is not returning a `CASAuthenticatedUser` instance - in fact, the CAS server does not know anything about the `CASAuthenticatedUser` class.  So, a more accurate comment would be:

> A CASAuthenticatedUser instance containing the ticket data returned by the CAS server.

Since the ticket _is_ what the CAS server returns.

Reply:
Committed the suggested JavaDoc change to the description of the return value. However, my latest commits are not showing up here after rebasing on staging/1.3.0 so maybe I need to go back to git school.